utils/apiUtil.js

```diff
@@ -403,16 +403,24 @@ module.exports = {
         if (process.env.DEBUG === 'true') {
           console.log(`#### ### ## podNameDebug: ${podNameDebug}`);
         }
-        const podName = await bash.runCommand(`g3kubectl get pod --sort-by=.metadata.creationTimestamp -l app=${labelName} -o jsonpath="{.items[*].metadata.name}" | awk "{print $NF}"`);
-        console.log(`latest pod found: ${podName}`);
+
+        // get all pods that match the provided label
+        const podsNames = await bash.runCommand(`g3kubectl get pod --sort-by=.metadata.creationTimestamp -l app=${labelName} -o jsonpath="{.items[*].metadata.name}" | awk "{print $NF}"`);
```

Comment:
won't `awk "{print $NF}"` filter the values and return the latest pod's name (since we are sorting the pods by creation time)?

Reply:
it's returning what you can see in the logs i copied in the PR description - a list of pod names

Reply:
but even if it worked, if the latest pod in the list is not the job we want, that's another bug 🤔

Reply:
just tested in qa-covid19:
```
$ g3kubectl get pod --sort-by=.metadata.creationTimestamp -l app=gen3job -o jsonpath="{.items[*].metadata.name}" | awk "{print $NF}"
usersync-hblkw arborist-rm-expired-access-27589875-bddlk arborist-rm-expired-access-27589880-r58t8
```

Reply:
looks like the issue is with the double quotes
```
qa-covid19@cdistest_dev_admin:~$ g3kubectl get pod --sort-by=.metadata.creationTimestamp -l app=gen3job -o jsonpath="{.items[*].metadata.name}"
arborist-rm-expired-access-27589895-5t7zt arborist-rm-expired-access-27589900-wshgr usersync-shl2n usersync-lk9tkqa-covid19@cdistest_dev_admin:~$ g3kubectl geqa-covid19@cdistest_dev_admin:~$ g3kubectl get pod --sort-by=.metadata.creationTimestamp -l app=gen3job -o jsonpath="{.items[*].metadata.name}" | awk '{print $NF}'
usersync-lk9tk
qa-covid19@cdistest_dev_admin:~$ g3kubectl get pod --sort-by=.metadata.creationTimestamp -l app=gen3job -o jsonpath="{.items[*].metadata.name}" | awk "{print $NF}"
arborist-rm-expired-access-27589895-5t7zt arborist-rm-expired-access-27589900-wshgr usersync-lk9tk
```

Reply:
and we use this like this (with the job-name):
```
await checkPod(I, 'metadata-aggregate-sync', 'gen3job,job-name=metadata-aggregate-sync', { nAttempts: 30, ignoreFailure: false, keepSessionAlive: false });
```